File: Kattis/Java/Tarifa.java
package tarifa;

import java.util.Scanner;

public class Tarifa {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int megabytes = sc.nextInt();
        int months = sc.nextInt();
        int allotted, total = 0;
        allotted = (megabytes * months) + megabytes; 
        
        for (int i = 0; i < months; i++) {
            total += sc.nextInt();
        }
        
        System.out.println((allotted - total));        
    }
    
}

File: Kattis/CPlusPlus/Tarifa.cpp
#include <iostream>

int main() {
    int megabytes, months, allotted, sum, total;

    std::cin >> megabytes >> months;
    allotted = ((megabytes * months) + megabytes);

    for (int i = 0; i < months; i++) {
        std::cin >> total;
        sum += total;
    }

    std::cout << (allotted - sum) << std::endl;

    system("PAUSE");
    return 0;
}